### PR TITLE
Refactoring and added codeclimate

### PR DESCRIPTION
**Pull request by Schultzer**
Okay. so we need to figure out the rules we should use in our ~~`.eslintrc`, `.codeclimate.yml` maybe CSSlint?~~
so we can figure out what we need to refactor and what is not necessary I need @danschultzer your 👀 on that task. 
-  ~~Ruleset for `.eslintrc` and `.codeclimate.yml`~~
- [x] Removed `.eslintrc` and `codeclimate.yml`
- [x] Refactoring
- [x] Indention
- [x] Single quotetation marks `'string'` for strings in our code base
-  ~~Move app menu from `background.js` to it's own `app_menu_template.js`~~

### CODE REVIEW

**Comment by danschultzer** [comment]
Remove code climate and eslint files, and remove codecliamte from readme.

File: src/background.js [hunk 1]
```diff
@@ -143,21 +143,39 @@ if (env.name !== 'production') {
     app.setPath('userData', userDataPath + ' (' + env.name + ')');
 }
 
+function openWindow(windowName, parentWindow) {
```
Comment (danschultzer) [inline src/background.js]: Why not just make this opts in the first place instead of using `parentWindow`?

File: src/file_list/file_list.spec.js [hunk 2]
```diff
@@ -260,117 +260,117 @@ describe("fileList", function() {
...
+      var expected = 'Name\tAmount\tDate\tPath\n' +
+        'csv-test.jpg\t500.00\t2016-01-05\t/path/to/csv-test.jpg\n' +
+        'csv-test-2.jpg\t\t\t/path/to/csv-test-2.jpg\n';
 
-      assert.equal(expected, fileList.toCSV(files));
+      assert.equal(fileList.toCSV(files), expected);
     });
 
-    it("handles \\t in values", function() {
+    it('handles \\t in values', function() {
```
Comment (danschultzer) [inline src/file_list/file_list.spec.js:335]: `\\t` should be changed to `\t`?

File: src/background.js (hunk 1 above)
Comment (danschultzer) [reply src/background.js]: Actually, all this should be refactored more.

**Comment by danschultzer** [comment]
Refactored openWindow method. The rest looks fine.

**Comment by danschultzer** [comment]
This is a massive change PR, so we need to merge this in fast. However, we should take some time to see if there's anything that needs to be refactored or cleaned up.

File: src/file_list/file_list.spec.js (hunk 2 above)
Comment (Schultzer) [reply src/file_list/file_list.spec.js:335]: No. We want to escape the tab in the string for readability.

**Pull request merged**